content/changelog/2022-10-04-console-release/index.md

```diff
@@ -0,0 +1,15 @@
+---
+label: 'Console'
+---
+
+### What's new
+
+- UI: Social authentication with Hasura provider.
+- Control Plane: Improved performance of project creation.
+
+### Bug fixes
+
+- UI: Database selector allows to select any of created databases.
+- Control Plane: Availability checker waits for existing operations queue for a project to be empty before starting.
+- Control Plane: Selecting operations for redo based on correct status.
+- Control Plane: Branch creation endpoint V2 is accessible via oauth token.
```

Comment:
```suggestion
- The V2 branch creation endpoint is now accessible using an OAuth token.
```

Reply:
HI Anton, 
The file structure for release notes just changed. This file will have to be moved to the appropriate location. Apologies for the inconvenience. Please see: https://github.com/neondatabase/website/pull/206#.